assembl/static2/js/app/components/debate/common/thematicTabs.jsx

```diff
@@ -45,12 +49,17 @@ const ThematicTabs = ({ guidelinesContent, semanticAnalysisForThematicData }: Pr
     </Tab>
   );
 
-  // Uncomment those line below when adding the summary
-  // const summaryTabAndContent = (
-  //   <Tab eventKey={ANNOUNCEMENT_TAB_ITEM_ID.SUMMARY} title={summaryTabTitle}>
-  //     {/** Summary Component and Content */}
-  //   </Tab>
-  // );
+  const summaryTabAndContent = (
+    <Tab eventKey={ANNOUNCEMENT_TAB_ITEM_ID.SUMMARY} title={summaryTabTitle}>
+      {summary ? (
+        <div className="announcement">
+          <div className="announcement-media announcement-text-only">
+            {activeHtml(summary && (isMobile ? summary : addIframeForMindManager(summary)), postBodyReplacementComponents())}
```

Comment:
So mobile mode doesn't show the iframe mindmap?

Reply:
nope because when a user scroll to the mind map, the scroll can be stuck inside preventing user to read further below the post